lib/elasticband/aggregation/max.rb

```diff
@@ -0,0 +1,9 @@
+module Elasticband
+  class Aggregation
+    class Max < FieldBased
+      def type
+        :max
```

Comment:
Not in Ruby 2.2 :)

Symbols are now garbage collected, a simple `freeze` will do the work